cli/azd/pkg/output/ux/list_as_text.go

```diff
@@ -14,7 +14,7 @@ import (
 // ["foo", "bar"] 					=> "foo and bar"
 // ["foo", "bar", "axe"] 			=> "foo, bar and axe"
 // ["foo", "bar", ..., ..., "axe"] 	=> "foo, bar, ..., ... and axe"
-func ListAsText(items []string) string {
+func listAsText(items []string, connectWord string) string {
```

Comment:
mind updating the comments above for the method?  You might want to move the docs to `AndListAsText` .
Then add docs for `OrListAsText`.

Optionally, add some docs for `listAsText`  (since this is not public)